Documentation/docs-site/content/usage/environments.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Environment"
+draft: false
+weight: 42
+---
+
+### Create an environment
+
+You can create an environment from node-env image and specify the resources such as memory and CPU along with poolsize. The poolsize controls how many warm pods are created initially.
+
+```
+fission env create --name node --image fission/node-env:0.4.0 --mincpu 40 --maxcpu 80 --minmemory 64 --maxmemory 128 --poolsize 4
+```
+
+In case of pool based executor, the resources specified for environment are used for function pod as well. In case of new deployment executor, you can override the resources when you create a function.
+
+### Using a builder
+
+When you create an environment, you can specify a builder image and builder command which will be used for building from source code. You can override the build command when creating a function. For more details on builder and packages you should check out examples in [Functions](../functions) and [packages](../package)
+
+```
+fission env create --name python --image gcr.io/fission-ci/python-env:test --builder gcr.io/fission-ci/python-env-builder:test
```

Comment:
We shouldn't be telling people to use our test builds... this guide should point at released images only.

Reply:
Yes, I had not found official builder image when I wrote this, fixed now.